packages/code-studio/src/index.tsx

```diff
@@ -1,29 +1,25 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
 import 'fira';
 import '@deephaven/components/scss/BaseStyleSheet.scss';
-import { MonacoUtils } from '@deephaven/console';
-import { store } from '@deephaven/redux';
-import MonacoWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
-import AppRouter from './main/AppRouter';
-import DownloadServiceWorkerUtils from './DownloadServiceWorkerUtils';
+import { LoadingOverlay } from '@deephaven/components';
+import { ApiBootstrap } from '@deephaven/jsapi-bootstrap';
 import logInit from './log/LogInit';
-import { unregister } from './serviceWorker';
 
 logInit();
 
+const AppRoot = React.lazy(() => import('./AppRoot'));
+
 ReactDOM.render(
-  <Provider store={store}>
-    <AppRouter />
-  </Provider>,
+  <ApiBootstrap
+    apiUrl={`${import.meta.env.VITE_CORE_API_URL}/${
+      import.meta.env.VITE_CORE_API_NAME
+    }`}
+    setGlobally
+  >
```

Comment:
Yea the complication with that is that in `AppRoot` it has an import of `@deephaven/console`, which imports `@deephaven/jsapi-shim` - so before we could even call `useContext` within the rendering of `AppRoot`, it would blow up.